exercises/anagram/test/test_anagram.c

```diff
@@ -76,6 +76,22 @@ static void test_detect_simple_anagram(void)
 
 }
 
+static void test_detect_adversarial_sum_anagram(void)
+{
+   char inputs[][MAX_STR_LEN] = {
+      "abcde"
+   };
+
+   char word[] = { "aacdf" };
+   candidates = build_candidates(*inputs, sizeof(inputs) / MAX_STR_LEN);
+   enum anagram_status expected[] = { NOT_ANAGRAM };
+
+   TEST_IGNORE();
```

Comment:
Per my comment in #458, can you put this into the "normal" spot? Looks good otherwise.

Reply:
Done!